modules/timepoint_list/php/timepoint_list_controlpanel.class.inc

```diff
@@ -64,13 +64,12 @@ Class TimePoint_List_ControlPanel extends \Candidate
         $this->tpl_data['candID'] = $this->getData('CandID');
 
         $this->tpl_data['isDataEntryPerson']
-            = $user->hasCenterPermission("data_entry", $cand_CenterID)
-            || $user->isUserDCC();
+            = $user->hasCenterPermission("data_entry", $cand_CenterID);
```

Comment:
Code looks good. Tagging as Passed Code Review. I don't see the relationship between the code modifications and the PR title though.

Reply:
It was checking if the user has permission at all, but the test plan says it should only display it for candidates at the same site, so it needed to be changed to be center permission. The code in the javascript/template was already there, the bool was just flagged based on the wrong thing.